### ISSUE TITLE
define our sound inventory 

### ISSUE THREAD

**Issue by masterkrang** [masterkrang/kako#5]

**Comment by masterkrang** [comment]
My idea for calculating the sound inventory (the sound palette / phonetics of kako) are like this:

Note: feedback welcome.

we aim to build our language with the most used sounds across all languages. this will reduce the learning curve across all people and all languages somewhat. i don't think it's a good idea to introduce a language with sounds that don't already exist, i think it will increase difficulty. yes, this is an assumption.

so going on this assumption, we have already determined through 3rd party research that these consonant and vowel sounds are the most used across all languages in this discussion https://github.com/masterkrang/kako/issues/6#issuecomment-149383852.

consonant: m k j p w b h g N ? n s tS S t f l "n "t nj
in languages: 425 403 378 375 332 287 279 253 237 216 202 196 188 187 181 180 174 160 152 141
frequency: 94.2 89.4 83.8 83.2 73.6 63.6 61.9 56.1 52.6 47.9 44.8 43.5 41.7 41.5 40.1 39.9 38.6 35.5 33.7 31.3

vowel: i a u E "o "e O o e a~
in languages: 393 392 369 186 181 169 162 131 124 83
frequency: 87.1 86.9 81.8 41.2 40.1 37.5 35.9 29.0 27.5 18.4

source: https://github.com/masterkrang/kako/issues/6#issuecomment-148963279

 given that these things are true, we need to keep the language size down, as one of our goals is to make the language at least 10% the size of other languages. so we should start with a very limited sound palette that will meet our short term goals. this should be **deflationary** much like bitcoin is a deflationary currency. nobody should be able to add these sounds without following a formula, and this formula is what we should strive to establish here.

here is how i think we can come up with the formula:

first determine the phonotactic structure (V / VC / CVC) that we feel should be suitable for the language. this will mathematically determine how many root words we can have (Phonotactic structure + # of syllables = number of root words)

once the **number of root words** that we feel is suitable has been decided upon, we could then easily pick and choose from our sound palette which ones to choose, **starting from most used sound**.

In summary:
- need to first determine phonotactic structure (V / CV / CVC) and total target number of root words.
- sounds are only preliminary and will likely grow in the future based on need. better to have too little to start, than too many.
- sounds introduced should follow a strict formula and be fully justified. arbitrarily introducing sounds into the language could have a negative effect like inflation in a monetary system.
- sounds are chosen from our sound palette starting with most used sound first.

cc @JoseUgalde @yhdaphne
